src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1014,34 +1014,34 @@
+    <value>Remove the call to 'System.Type.InvokeMember' with 'BindingFlags.NonPublic' from '{0}'. Taking a dependency on a private member increases the chance of a breaking change in the future.</value>
   </data>
   <data name="AvoidCallingProblematicMethodsMessageCoInitializeSecurity" xml:space="preserve">
-    <value>{0} is a P/Invoke declaration to an OLE32 API that cannot be reliably called after the runtime has been initialized. The workaround is to write an unmanaged shim that will call the routine and then activate and call into managed code. You can do this using an export from a mixed-mode C++ DLL, by registering a managed component for use by COM, or by using the runtime hosting API.</value>
+    <value>'{0}' is a P/Invoke declaration to an OLE32 API that cannot be reliably called after the runtime has been initialized. The workaround is to write an unmanaged shim that will call the routine and then activate and call into managed code. You can do this using an export from a mixed-mode C++ DLL, by registering a managed component for use by COM, or by using the runtime hosting API.</value>
   </data>
   <data name="AvoidCallingProblematicMethodsMessageCoSetProxyBlanket" xml:space="preserve">
-    <value>{0} is a P/Invoke declaration to an OLE32 API that cannot be reliably called against a runtime callable wrapper (a managed object wrapping a COM object). Runtime callable wrappers dynamically fetch interface pointers so the effect of the call might be arbitrarily lost. Runtime callable wrappers for a given COM object are also shared across an application domain so the call could possibly affect other users. Replace this call with a native wrapper COM object for the interface pointer that does the appropriate CoSetProxyBlanket calls.</value>
+    <value>'{0}' is a P/Invoke declaration to an OLE32 API that cannot be reliably called against a runtime callable wrapper (a managed object wrapping a COM object). Runtime callable wrappers dynamically fetch interface pointers so the effect of the call might be arbitrarily lost. Runtime callable wrappers for a given COM object are also shared across an application domain so the call could possibly affect other users. Replace this call with a native wrapper COM object for the interface pointer that does the appropriate CoSetProxyBlanket calls.</value>
   </data>
   <data name="AvoidCallingProblematicMethodsMessageSystemRuntimeInteropServicesSafeHandleDangerousGetHandle" xml:space="preserve">
-    <value>Remove the call to SafeHandle.DangerousGetHandle from {0}</value>
+    <value>Remove the call to 'SafeHandle.DangerousGetHandle' from '{0}'</value>
   </data>
   <data name="AvoidCallingProblematicMethodsMessageSystemReflectionAssemblyLoadFrom" xml:space="preserve">
-    <value>Remove the call to Assembly.LoadFrom from {0}</value>
+    <value>Remove the call to 'Assembly.LoadFile' from '{0}'</value>
```

Comment:
Message was `LoadFrom` and you changed it to `LoadFile`.

Reply:
🤯

Reply:
@Evangelink Copy/paste error 😕 

Good catch!
I'll review all changes again, and address feedbacks from you and @sharwell.